src/stories/20_Features/Animations.html

```diff
@@ -258,57 +258,61 @@ <h1>zoom-out</h1>
 
     <!-- ## Pan-Left -->
     <amp-story-page id="pan-left">
-      <amp-story-grid-layer template="vertical">
-        <h1>pan-left</h1>
-        <amp-img animate-in="pan-left"
-                 animate-in-duration="4s"
-                 layout="fixed"
-                 src="https://picsum.photos/1024/768?image=1026"
-                 width="720"
-                 height="320">
-        </amp-img>
+      <amp-story-grid-layer template="fill">
```

Comment:
Can you re-add the title in the second layer?

Reply:
Sure 👍